discord_thread_summarizer_using_openai/main.bal

```diff
@@ -0,0 +1,79 @@
+import ballerina/io;
+import ballerina/time;
+
+const string DISCORD_API_URL = "https://discord.com/api/v9";
```

Comment:
```suggestion
const DISCORD_API_URL = "https://discord.com/api/v9";
```

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/1dbf76e067fd08cc0430182aa02b7c98d154f3ac